WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/lists/sections/viewholders/BlockListItemViewHolder.kt

```diff
@@ -42,6 +43,15 @@ open class BlockListItemViewHolder(
         }
     }
 
+    protected fun TextView.setTextOrHide(item: BlockListItem.TitleWithMore) {
+        this.visibility = View.VISIBLE
+        if(item.navigationAction == null){
+            this.ellipsize = null
+            this.maxLines = Int.MAX_VALUE
+        }
+        this.setTextOrHide(item.textResource,item.text)
+    }
+
```

Comment:
- Nice! It's good to remove ellipsize end making the text view multiline. 👍🏻 
- I think the function name could be better. This function doesn't only set or hide the text view. It has a specific logic inside.
- Also, this function has a parameter of  `BlockListItem.TitleWithMore` type, so it'll be used only in `TitleWithMore`. We can move this function to `TitleWithMore`. I don't think we can benefit from having this function in `BlockListItemViewHolder`.

Reply:
Thank you for reviewing the PR 🙌. 
Will do the changes as requested and push it.